crates/astria-sequencer/src/transaction/action.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::Result;
+use astria_proto::sequencer::v1::{
+    action::Value::{
```

Comment:
Since these enum variants only show up in function bodies below, you can just import them locally so you don't have to pollute your module namespace. I.e. smth like (changed the type alias to be on `Value` so that not every variant has to be renamed):
```rust
fn my_fn() {
    use astria_proto::sequencer::v1::action::Value as ProtoValue;
    match v {
        ProtoValue::SequenceAction ...
   }
}
```